karma.conf.js

```diff
@@ -0,0 +1,69 @@
+// Karma configuration
+// Generated on Tue Mar 31 2015 23:46:42 GMT+0200 (CEST)
+
+module.exports = function (config) {
+    config.set({
+
+        // base path that will be used to resolve all patterns (eg. files, exclude)
+        basePath: '',
+
+
+        // frameworks to use
+        // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+        frameworks: ['qunit'],
+
+
+        // list of files / patterns to load in the browser
+        files: [
+            'node_modules/jquery/dist/jquery.js',
+            //'js/**/*.js',
+            'dist/jquery.inputmask.bundle.js',
```

Comment:
We could possibly look into always having the UMD wrapper in the source, and not injecting at build time. Then using e.g. https://github.com/Nikku/karma-browserify.
Then we could have the real source tested, and not have to fetch jquery manually from `node_modules`